supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/CondorJobExecutionActor.scala

```diff
@@ -0,0 +1,195 @@
+    props(jobDescriptor, configurationDescriptor, htCondorCmds, htCondorParser, htCondorProcess)
+
+  def props(jobDescriptor: BackendJobDescriptor,
+            configurationDescriptor: BackendConfigurationDescriptor,
+            cmds: HtCondorCommands,
+            parser: HtCondorClassAdParser,
+            extProcess: HtCondorProcess): Props =
+    Props(new CondorJobExecutionActor(jobDescriptor, configurationDescriptor, cmds, parser, extProcess))
+}
+
+class CondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor,
+                              override val configurationDescriptor: BackendConfigurationDescriptor,
+                              cmds: HtCondorCommands,
+                              parser: HtCondorClassAdParser,
+                              extProcess: HtCondorProcess) extends BackendJobExecutionActor with SharedFileSystem {
+
+  import CondorJobExecutionActor._
+  import cromwell.core.PathFactory._
+  import better.files._
+
+  val fileSystemsConfig = configurationDescriptor.backendConfig.getConfig("filesystems")
+  override val sharedFsConfig = fileSystemsConfig.getConfig("local")
+
+  val workflowDescriptor = jobDescriptor.descriptor
+  val jobPaths = new JobPaths(workflowDescriptor, configurationDescriptor.backendConfig, jobDescriptor.key)
+
+  // Files
+  val executionDir = jobPaths.callRoot
+  val returnCodePath = jobPaths.returnCode
+  val stdoutPath = jobPaths.stdout
+  val stderrPath = jobPaths.stderr
+  val scriptPath = jobPaths.script
+  val submitPath = jobPaths.submitFile
+
+  lazy val stdoutWriter = extProcess.getUntailedWriter(stdoutPath)
+  lazy val stderrWriter = extProcess.getTailedWriter(100, stderrPath)
+  val argv = extProcess.getCommandList(scriptPath.toString)
+
+  val call = jobDescriptor.key.call
+  val callEngineFunction = CondorJobExpressionFunctions(jobPaths)
+
+  val lookup = jobDescriptor.inputs.apply _
+
+
+  val runtimeAttributes = {
+    val evaluateAttrs = call.task.runtimeAttributes.attrs mapValues (_.evaluate(lookup, callEngineFunction))
+    // Fail the call if runtime attributes can't be evaluated
+    TryUtil.sequenceMap(evaluateAttrs, "Runtime attributes evaluation").get.mapValues(_.valueString)
+  }
+
+  /**
+    * Restart or resume a previously-started job.
+    */
+  override def recover: Future[BackendJobExecutionResponse] = Future.failed(new Exception("HtCondorBackend currently doesn't support recovering jobs."))
+
+  /**
+    * Execute a new job.
+    */
+  override def execute: Future[BackendJobExecutionResponse] = Future(executeTask())
+
+  /**
+    * Abort a running job.
+    */
+  override def abortJob(): Unit = Future.failed(new Exception("HtCondorBackend currently doesn't support aborting jobs."))
+
+  override def preStart(): Unit = {
+    log.debug(s"Creating execution folder: $executionDir")
+    executionDir.toString.toFile.createIfNotExists(true)
```

Comment:
They actually say to use `log.debug("blah blah {}", foo)`
I feel like `log.debug("blah blah " + foo)` would be evaluated too no ?